Content.Server/Polymorph/Systems/PolymorphSystem.cs

```diff
@@ -365,8 +370,8 @@ private void OnDestruction(Entity<PolymorphedEntityComponent> ent, ref Destructi
         if (TryComp<PolymorphableComponent>(parent, out var polymorphableComponent))
             polymorphableComponent.LastPolymorphEnd = _gameTiming.CurTime;
 
-        // if an item polymorph was picked up, put it back down after reverting
-        _transform.AttachToGridOrMap(parent, parentXform);
+        if (TryComp<BeingCarriedComponent>(uid, out var carried))
+            _carrying.DropCarried(carried.Carrier, uid);
```

Comment:
This doesn't account for the situation where the polymorphed or the held entity is an item though. This may result in players getting sent into nullspace.

Reply:
got it, that explains why the separate line about carried items was there. added back the line, and additionally to the initial polymorph in case someone is an item in the first place